src/app/app-routing.module.ts

```diff
@@ -21,11 +21,16 @@ import {
   SignUpComponent
 } from 'sign-up/sign-up.component';
 
+import { 
+  AuthGuardService 
+} from 'shared/services/auth-guard.service';
+
 
 const routes: Routes = [
   {
     path: '',
     redirectTo: '/log-in',

```

Comment:
Wondering about this... no path means you either go to log-in (if local storage is empty) or select-course (if storage has values). So yeah.... naa jud dapat ang log-in redirect here? Just curious.

Reply:
I will polish it later, though routes could not be modified by guards, only paths.